src/st/entity_stage_name_popup_jp.h

```diff
@@ -326,7 +331,8 @@ void EntityStageNamePopup(Entity* self) {
             }
             break;
         case 3:
-            PrimDecreaseBrightness(self->ext.stpopupj.unk88, 4);
+            prim = self->ext.stpopupj.unk88;
+            PrimDecreaseBrightness(prim, 4);
```

Comment:
This change was required on NO4 but does not break existing PSP usages of the header, so I suspect it is probably the canonical implementation.